### ISSUE TITLE
Unable to add parameters in Email links

### ISSUE THREAD

**Issue by marcwalsh** [MozillaFoundation/foundation.mozilla.org#10131]
When using either External Links or the Email links in Wagtail editor, I’m unable to add subject and body parameters to auto fill an email:

An example link looks likes this:

mailto:marc@example.com?subject=This%20is%20the%20subject&body=This%20is%20the%20body%20of%20the%20email

When trying to enter this as an external link, or adding the parameters on the email address, both fail validation.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/TP1-125)

**Comment by mtdenton** [comment]
@marcwalsh Selecting this for grooming session today to look at paths forward.

**Comment by mtdenton** [comment]
@marcwalsh can you provide us a page where this is happening and steps to reproduce this issue? Specifically, what field are you filling out that you're seeing failed validation?

**Comment by marcwalsh** [comment]
We weren't able to publish but it's on any MozFest Primary Page when adding a link using the text editor:

<img width="1381" alt="Screenshot 2023-03-08 at 18 41 21" src="https://user-images.githubusercontent.com/30226684/223805541-23e90be1-e6d9-4c62-8ce1-f016b9e6c000.png">
<img width="1377" alt="Screenshot 2023-03-08 at 18 42 09" src="https://user-images.githubusercontent.com/30226684/223805701-aeadec27-f11d-4cda-ad31-deca0ac22f9c.png">

However adding the following to a link button works as expected:
`mailto:marc@example.com?subject=This%20is%20the%20subject&body=This%20is%20the%20body%20of%20the%20email` where it opens the default email editor with the address, subject and body already filled in.

**Comment by data-sync-user** [comment]
➤ Simon Acosta Torres commented:

Tyler Denton This was initially reported by Marc on March 2023, but hasn’t pop up again. Would suggest as a candidate for removal till we have a similar issue reported.

**Comment by marcwalsh** [comment]
Hey - I just tested and I'm not able to include parameters in email links (I tried both mailto: and just the email with the parameters).

<img width="932" alt="Screenshot 2024-06-19 at 19 56 37" src="https://github.com/MozillaFoundation/foundation.mozilla.org/assets/30226684/ac7110fe-aea7-4c12-a553-be1f068ec7f1">

**Comment by simont-cr** [comment]
Hi @marcwalsh , while I'm sorry to hear that this is still an issue, I also thank you for such a quick response to verify. He will up the priority of this bug to make sure we get this resolved soon.

**Comment by data-sync-user** [comment]
➤ Mavis Ou commented:

After some digging I found out that in Wagtail v5.2 they added support for email chooser form to include subject and body to mailto links (PR here ( https://github.com/wagtail/wagtail/pull/10920 ) & changelog here ( https://docs.wagtail.org/en/stable/releases/5.2.html#other-features )). I also found some doc on how to implement the email chooser form here ( https://docs.wagtail.org/en/v5.2.2/reference/contrib/forms/index.html#form-builder ) . However, we haven't implemented any Wagtail forms on our site, so I'd like to check with the team if it's something we want to adopt. I can bring this up in our next group meeting / Slack next week.

/cc Simon Acosta Torres
